jimbopulos/ecothot
# Add a NavBar

## Addresses
[Add a NavBar - Asana]( https://app.asana.com/0/1202639946826739/1206089289277090/f)

## Summary
- Introduces new Next routes
- Creates a reusable `NavBar` component
- Basic styles

## Testing
- clone this repo
- open command line at project root (`ctrl + ~`)
- `npm run dev`
- open local host server at `http://localhost:3000/`
- Observe `NavBar` at top center and expected functionality:
  - Proper routing for each nav item
  - persistence of NavBar on each page
 
<img width="1256" alt="Screenshot 2024-01-08 at 5 17 51 PM" src="https://github.com/jimbopulos/ecothot/assets/76788080/92c41905-82c0-4b5c-b050-68c52f61a44a">
